From f5f86dbc45ff1f651ca8065346397406bd3512e6 Mon Sep 17 00:00:00 2001
From: HungTran <hung.tran.jy@rvc.renesas.com>
Date: Fri, 19 Feb 2016 15:12:36 +0700
Subject: [PATCH] Update new path for simple player

One place in simple player still uses the old path (/usr/share/)
Change it to new path (/home/root/) for consistent with current system

Signed-off-by: HungTran <hung.tran.jy@rvc.renesas.com>
---
 qml/simpleplayer/main.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qml/simpleplayer/main.qml b/qml/simpleplayer/main.qml
index 0fe7981..78bc60f 100755
--- a/qml/simpleplayer/main.qml
+++ b/qml/simpleplayer/main.qml
@@ -4464,7 +4464,7 @@ Rectangle {
             id: folderModel1
             showFiles : true
             showHidden: true
-            folder: "file:///usr/share/doorphone"
+            folder: "file:///home/root/doorphone"
             nameFilters: ["show_icon_flag"]
             showDirs : false
 
-- 
1.9.1

